extensions/resource-deployment/src/ui/modelViewUtils.ts

```diff
@@ -731,31 +745,66 @@ async function processAzureAccountField(context: AzureAccountFieldContext): Prom
 	context.fieldInfo.subFields = [];
 	const accountValueToAccountMap = new Map<string, azdata.Account>();
 	const subscriptionValueToSubscriptionMap = new Map<string, azureResource.AzureResourceSubscription>();
-	const accountDropdown = createAzureAccountDropdown(context);
+	const accountComponents = createAzureAccountDropdown(context);
+	const accountDropdown = accountComponents.accountDropdown;
 	const subscriptionDropdown = createAzureSubscriptionDropdown(context, subscriptionValueToSubscriptionMap);
 	const resourceGroupDropdown = createAzureResourceGroupsDropdown(context, accountDropdown, accountValueToAccountMap, subscriptionDropdown, subscriptionValueToSubscriptionMap);
+	if (context.fieldInfo.allowNewResourceGroup) {
```

Comment:
Did you run this by design? I really don't like that we have two fields for RG - what changes when you check the checkbox? It seems like we should just have the single editable dropdown and just make it clear that a new RG will be created if the name doesn't exist (maybe have a conditional text field that only shows up if the currently entered name isn't in the list of RGs)

Reply:
<img width="929" alt="Screen Shot 2020-07-06 at 1 57 18 PM" src="https://user-images.githubusercontent.com/13777222/86654051-84141980-bf9a-11ea-9756-63ac8f567eea.png">

Reply:
Alrighty, my opinion is that the checkbox should control what controls are enabled/disabled at least then. So when unchecked dropdown is enabled and text box is disabled. And vice versa for checked.

Reply:
yep, it is already implemented that way.